src/app/preferences/_preferences.html

```diff
@@ -8,7 +8,7 @@
 		{{'PREFERENCES_MANAGE_SETTINGS_PREFERENCES' | translate}}<br />
 	</p>
 	
-    <div class="menu">
+    <!--<div class="menu">
```

Comment:
Actually, yes, you want. ;)
We are not going to show an empty page or a menu page. So, we simply display the main preference page (which is the language one). This may change later, once we have more pages.